project_generator/templates/makefile_gcc.tmpl

```diff
@@ -44,7 +44,10 @@ LD_SCRIPT = {{linker_file}}
 
 CC_SYMBOLS = {% for symbol in macros %} -D{{symbol}} {% endfor %}
 
-LIBS = {% for library in source_files_lib %} -l{{library}} {% endfor %}
+LIBS = {% for library in libraries %} {{library}} {% endfor %}
+LIB_PATHS = {% for path in lib_paths %} {{path}} {% endfor %}
+LD_SYS_LIBS = -lstdc++ -lsupc++ -lm -lc -lgcc -lnosys
```

Comment:
Yes, a user should. As if someone does not use c++, then first 2 does not make sense for example.